pkg/attestation/crafter/runner.go

```diff
@@ -41,6 +42,12 @@ type SupportedRunner interface {
 
 	// ID returns the runner type
 	ID() schemaapi.CraftingSchema_Runner_RunnerType
+
+	// WorkflowFile returns the workflow file associated with this runner
+	WorkflowFile(ctx context.Context) string
```

Comment:
we might need to be more explicit of what this should return, is this a reference? a path? the content?

Reply:
Renamed, done.